nifi-nar-bundles/nifi-groovyx-bundle/nifi-groovyx-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -10,8 +10,42 @@ Apache Software License v2
 
 The following binary components are provided under the Apache Software License v2
 
-Apache Groovy
-Copyright 2003-2015 The Apache Software Foundation
+  (ASLv2) Groovy (org.codehaus.groovy:groovy-all:pom:2.5.4 and its submodules included as transitive dependencies - http://www.groovy-lang.org)
```

Comment:
Appears to be missing the transitive dependencies that are brought in by the Groovy modules.

Reply:
Good catch.

I'll add the necessary NOTICE and LICENSE content for the transitive dependencies of Groovy that are bundled in the NAR.

Some notes:
- apiguardian-api, jcommander, opentest4j, picocli, TestNG, and qbox are all ASLv2, and contain no NOTICE of their own.
- ant and commons-cli are under ASLv2, and provide their own NOTICE
- jLine has a BSD LICENSE, no NOTICE of its own.
- junit-platform and junit-jupiter modules are under EPLv2, and contain no NOTICE of their own.